src/main.rs

```diff
@@ -341,6 +380,16 @@ fn run() -> Result<(), Box<::std::error::Error>> {
                 .value_name("N")
                 .help("An alias for -n/--length"),
         )
+        .arg(
+            Arg::with_name("nosqueeze")
+                .short("v")
+                .long("no-squeeze")
```

Comment:
Can we be consistent with `hexdump` and call this `--no-squeezing`?

Reply:
certainly!